omnigibson/robots/tiago.py

```diff
@@ -697,11 +698,37 @@ def arm_workspace_range(self):
             "right": [np.deg2rad(-75), np.deg2rad(-15)],
         }
 
-    def get_position_orientation(self):
-        # TODO: Investigate the need for this custom behavior.
+    def get_position_orientation(self, frame: Literal["world", "scene", "parent"] = "world"):
+        """
+        Gets tiago's pose with respect to the specified frame.
+
+        Args:
+            frame (Literal): frame to get the pose with respect to. Default to world. parent frame
+            get position relative to the object parent. scene frame get position relative to the scene.
+
+        Returns:
+            2-tuple:
+                - 3-array: (x,y,z) position in the specified frame
+                - 4-array: (x,y,z,w) quaternion orientation in the specified frame
+        """
+
         return self.base_footprint_link.get_position_orientation()
 
-    def set_position_orientation(self, position=None, orientation=None):
+    def set_position_orientation(
```

Comment:
Also need to be careful with Tiago, this robot also has a fixed joint to the world